OPHD/UI/TextRender.h

```diff
@@ -5,5 +5,5 @@
 #include <string>
 
 
-void drawLabelAndValue(NAS2D::Point<int> position, const std::string& title, const std::string& text, NAS2D::Color color = NAS2D::Color::White);
+void drawLabelAndValue(NAS2D::Point<int> position, const std::string& title, const std::string& text, int valuePadding = 0, NAS2D::Color color = NAS2D::Color::White);
```

Comment:
While less accurate, I wonder if this should be handled by adding spaces at the end of the `title` string. It's starting to seem like the method is becoming burdened with too many parameters.

Reply:
I put valuePadding here to allow for the current use cases in the code more than an endorsement of the idea. In an eventual string table approach, this would be replaced by a columnPadding property that would improve the multiple arguments in this function.

Looking at this closer, it doesn't seem to make sense why the padding exists. I think it would look better left justified. Maybe some judgement here, but I'll change to left justify to remove the padding use case and avoid the extra space solution.